app/src/main/java/net/simplifiedcoding/shelounge/ui/MapsActivity.java

```diff
@@ -247,7 +259,7 @@ public void onClick(View v) {
 
                         LatLngBounds GWALIOR = new LatLngBounds(new LatLng(26.104292, 78.111943), new LatLng(26.320862, 78.270518));
```

Comment:
mention the coordinates in app constants so we don't have to change at many places